massdriver.yaml

```diff
@@ -140,17 +141,6 @@ params:
             - properties:
                 zone_redundancy:
                   const: false
```

Comment:
Can you elaborate on the decision behind it? Geo-redundant storage and read-access geo-redundant storage may be options that a customer wants due to stuff like compliance requirements

Reply:
Sure thing :) 

- An account replication type is _required_ 
- Those options are [here](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/storage_account#account_replication_type)
- This parameters in the `massdriver.yaml` seems to enable _other_ types of replication. ZRS, GZRS, RAGZRS

If we want the user to select LRS, GRS, RAGRS, _or_ one of these other types, then maybe we can reorganize this a bit. Instead of `zone_redundancy` being a boolean, maybe we just give them the replication types to select?

Reply:
The problem with the replication types is what is allowed after you already deploy. If you deploy to a non-zonal type, like LRS, GRS, or RAGRS, then you can **only** switch between those 3. If you deploy to a zonal type like ZRS, GZRS, RAGZRS, then you can **only** switch between those 3 zonal types. If that makes sense. You can switch between zonal types if you're zonal and non-zonal replication types if you're non-zonal.